scf/README.md

```diff
@@ -1,48 +1,56 @@
+## Prerequisites:
 
 - Make sure your Kubernetes cluster meets all [SCF related Kubernetes Requirements](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#requirements-for-kubernetes).
 - Install [helm](https://helm.sh/)
 
 ## Deploy
 
 1. Choose a [non NFS based `StorageClass`](https://github.com/SUSE/scf/wiki/How-to-Install-SCF#choosing-a-storage-class) because MySQL does not work well with it. For additional information you can take a look at [Cloud Specifics](#cloud-specifics)
-1. Download and unzip the latest [release](https://github.com/cloudfoundry-incubator/eirini-release/releases)
-1. Create a `scf-config.yaml` file using the following template:
+1. Clone or download and unzip the latest  [Eirini Release](https://github.com/cloudfoundry-incubator/eirini-release/releases)
+1. Create a `scf-config-values.yaml` file under `eirini-release/scf/helm`, using the following template:
     ```yaml
     env:
+      # should be a `domain` name that resolves to any of your non-master cluster nodes
+      # alternatively, you can use `nip.io` and public IP for any of your non-master nodes
+      # in the form of `x.x.x.x.nip.io`
       DOMAIN: <cf-domain-address>
+      # `uaa.doamin` or `uaa.x.x.x.x.nip.io` as mentioned above
       UAA_HOST: <cf-uaa-address>
+      # default UAA_PORT is 2793
       UAA_PORT: <cf-uaa-port>
-      # To disable diego and use eirini/opi staging, set the following parameter to `true`:
+      # to disable diego and use eirini/opi staging, set the following parameter to `true`
+      # if set to `false`, diego cells are used to stage the droplet and create the application OCI image
       ENABLE_OPI_STAGING: false
 
     kube:
       auth: rbac
-      #List all the kube node ips
+      # list all the kube node ips
       external_ips:
       - <kube-node-ip>
       storage_class:
-      #depends on your specific storage class
-        persistent: hostpath
-        shared: hostpath
+        persistent: <storage-class>
+        shared: <storage-class>
 
     opi:
-      # The ingress sub-domain or IP
-      ingress_endpoint: <ingress-endpoint>
-
-      # The namespace eirini/opi schedules the apps to.
-      namespace: <kubernetes-namespace>
-
       # if this property is set to true it will expose the
       # registry via ingress, default is to NodePort.
       use_registry_ingress: false
 
-      # set to false if you don't want Eirini to create ingress rules for apps
-      use_app_ingress: true
+      # the ingress sub-domain or IP
+      # you can ignore this porperty if use_registry_ingress is set to `false`
+      ingress_endpoint: <ingress-endpoint>
+
+      # the namespace eirini/opi schedules the apps to.
+      namespace: <kubernetes-namespace>
+
+      # if you clone the eirini-release repo you need to explicitly specify the image tag.
+      # when downloading the release tarball, `image_tag` is already set to `latest`, in
+      # which case you can drop the following property.
+      image_tag: latest
```

Comment:
We actually started setting this to a particular value, but we'll fix this after merging the PR.